FacuReartes/react-laba-final-project-team-2
# SPT2-179 - Home Page Pagination

# Description


## Changes

- Added infinite scroll using [useInfiniteQuery](https://tanstack.com/query/latest/docs/framework/react/reference/useInfiniteQuery#useinfinitequery) in `HomePageContainer` component
- Fixed bug related to server side products prefetching where the products list shows old deleted products

### Jira

- [SPT2-179](https://solvd-final-project-team-2.atlassian.net/browse/SPT2-179?atlOrigin=eyJpIjoiYzRiYzIyNjFkZDZmNDc1NTg5ZGU2NTU2NDA1OTRmMGEiLCJwIjoiaiJ9)

### How to Test

1. Navigate to ['http://localhost:3000/'](http://localhost:3000/).
2. Open Navigation tab in devtools
3. In Home Page, scroll down products list
4. Check new page refetch in navigation tab

### Checklist



[SPT2-179]: https://solvd-final-project-team-2.atlassian.net/browse/SPT2-179?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
